.github/checkgroup.yml

```diff
@@ -233,27 +233,38 @@ subprojects:
       - "requirements/app/**"
       - "src/lightning_app/**"
       - "tests/tests_app/**"
-      - "tests/tests_app_examples/**"
-      # the examples are used in the app CI
-      - "examples/app_*"
+      - "examples/app_*"  # some tests_app tests call examples files
+      - ".github/workflows/ci-app-tests.yml"
+      - "setup.py"
       - ".actions/**"
     checks:
       - "App.cloud-e2e"
       - "make-doctest (app)"
       - "make-html (app)"
-      - "app-examples (macOS-11, app, 3.8, latest)"
```

Comment:
but they shall stay here, as if you change source it may affect examples

Reply:
They are already in https://github.com/Lightning-AI/lightning/pull/15332/files#diff-5ca41bd8852d2c309825fa76269fb299e7a77b075918a9c56fa1990c40e48f7bR261-R266. If you modify an examples file, that group will cover it

Reply:
@Borda are we good to resolve so that we can merge this?